just-ts-mode.el

```diff
@@ -332,8 +332,6 @@ The function removes existing entries for the Just language in
 (provide 'just-ts-mode)
 
 ;;;###autoload
-(progn
-  (add-to-list 'auto-mode-alist '("/[Jj]ustfile\\'" . just-ts-mode))
-  (add-to-list 'auto-mode-alist '("\\.[Jj]ust\\(file\\)?\\'" . just-ts-mode)))
+(add-to-list 'auto-mode-alist '("\\(?:\\.\\)?[Jj]ust\\(?:file\\)?\\'" . just-ts-mode))
```

Comment:
I think this regex will match files like `/a/b/just`, which the previous 2 regexes didn't. I didn't believe we should match that filename. Do you think we should?